File: specification/eventgrid/resource-manager/Microsoft.EventGrid/preview/2023-06-01-preview/examples-java/PartnerTopics_Delete.java
/** Samples for PartnerTopics Delete. */
public final class Main {
    /*
     * x-ms-original-file: specification/eventgrid/resource-manager/Microsoft.EventGrid/preview/2023-06-01-preview/examples/PartnerTopics_Delete.json
     */
    /**
     * Sample code: PartnerTopics_Delete.
     *
     * @param manager Entry point to EventGridManager.
     */
    public static void partnerTopicsDelete(com.azure.resourcemanager.eventgrid.EventGridManager manager) {
        manager.partnerTopics().delete("examplerg", "examplePartnerTopicName1", com.azure.core.util.Context.NONE);
    }
}

File: specification/eventgrid/resource-manager/Microsoft.EventGrid/preview/2023-12-15-preview/examples-js/PartnerTopics_Delete.js
const { EventGridManagementClient } = require("@azure/arm-eventgrid");
const { DefaultAzureCredential } = require("@azure/identity");

/**
 * This sample demonstrates how to Delete existing partner topic.
 *
 * @summary Delete existing partner topic.
 * x-ms-original-file: specification/eventgrid/resource-manager/Microsoft.EventGrid/preview/2023-12-15-preview/examples/PartnerTopics_Delete.json
 */
async function partnerTopicsDelete() {
  const subscriptionId =
    process.env["EVENTGRID_SUBSCRIPTION_ID"] || "8f6b6269-84f2-4d09-9e31-1127efcd1e40";
  const resourceGroupName = process.env["EVENTGRID_RESOURCE_GROUP"] || "examplerg";
  const partnerTopicName = "examplePartnerTopicName1";
  const credential = new DefaultAzureCredential();
  const client = new EventGridManagementClient(credential, subscriptionId);
  const result = await client.partnerTopics.beginDeleteAndWait(resourceGroupName, partnerTopicName);
  console.log(result);
}
